stripper/ze_Serpentis_Temple_p2.cfg

```diff
@@ -28,4 +28,62 @@ modify:
 	{
 		"OnStartTouch" "!activatorSetHealth00-1"
 	}
+}
+
+;Adding back warmup round. Why is it there if it isn't used?
+;Since the warmup end nuke doesn't exist (there is still an output that has no effect though) I will add a proper round end & invulnerability.
+modify:
+{
+	match:
+	{
+		"hammerid" "74390"
+		"origin" "-12112 -11270 6898"
+		"targetname" "Level_Counter"
+	}
+	replace:
+	{
+		"min" "1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"hammerid" "74392"
+		"classname" "logic_case"
+		"targetname" "Level_Case
+	}
+	insert:
+	{
+		"OnCase01" "Warmup_NoZombieTimer,Enable,,0.2,-1"
+		"OnCase01" "Warmup_EndRound,EndRound_Draw,5,60,-1"
+		"OnCase01" "Warmup_NoZombieTimer,Disable,,60,-1"
+		"OnCase01" "Warmup_NoZombieTimer,Kill,,60.2,-1"
+	}
+}
+
+add:
+{
+	"classname" "filter_activator_class"
+	"targetname" "Filter_NoPlayer"
+	"filterclass" "player"
+	"Negated" "1"
+}
+
+add:
+{
+	"classname" "logic_timer"
+	"targetname" "Warmup_NoZombieTimer"
+	"StartDisabled" "1"
+	"RefireTime" "1"
+	"spawnflags" "0"
+	
+	"OnTimer" "player,SetDamageFilter,Filter_NoPlayer,0,-1"
+}
+
+add:
+{
+	"classname" "game_round_end"
+	"targetname" "Warmup_EndRound"
 }
```

Comment:
Don't really see the point in explicitly adding a warmup honestly, and the reason it isn't enabled should be pretty obvious if you had to add all this to get it to work.